packages/lib/src/core/presets/magicGallery.js

```diff
@@ -2,35 +2,41 @@ import LAYOUTS from '../../common/constants/layout';
 import SCROLL_DIRECTION from '../../common/constants/scrollDirection';
 import { featureManager } from '../helpers/versionsHelper';
 
-export const fixedStyles = {
-  galleryLayout: LAYOUTS.MAGIC,
-  cubeImages: undefined,
-  cubeRatio: undefined,
-  isVertical: undefined,
-  targetItemSize: undefined,
-  collageAmount: undefined,
-  collageDensity: undefined,
-  groupTypes: undefined,
-  oneRow: undefined, // later on in layoutHelper this can be changed if it is false, so not exactly fixed.
-  imageMargin: undefined,
-  scatter: undefined,
-  galleryMargin: undefined,
-  chooseBestGroup: undefined,
-  smartCrop: undefined,
-  cubeType: undefined,
-  hasThumbnails: undefined,
-  enableScroll: undefined,
-  isGrid: undefined,
-  isSlideshow: undefined,
-  isSlider: undefined,
-  isColumns: undefined,
-  cropOnlyFill: undefined,
-  fixedColumns: undefined,
-  enableInfiniteScroll: undefined,
-  slideshowLoop: false,
+const fixToMagic = (styles) => {
+  let presetStyles = { ...styles };
+  presetStyles.galleryLayout = LAYOUTS.MAGIC;
+  presetStyles.cubeImages = undefined;
```

Comment:
I prefer:
`['cubeImages', 'cubeRatio', ...].forEach(p => delete presetStyles[p])`

Reply:
All I can do is change this to delete commands. (and I dont think its worth the change for now)
this about the next step where Ill need to delete / assign undefined to a.b.c.d.cubeSomething (as a deeper key)

Reply:
It's just a style suggestions to make it shorter. The PR is great :)